zone/gm_commands/find.cpp

```diff
@@ -37,6 +38,7 @@ void command_find(Client *c, const Seperator *sep)
 	std::vector<Cmd> commands = {
 		Cmd{.cmd = "aa", .u = "aa [Search Criteria]", .fn = FindAA, .a = {"#findaa"}},
 		Cmd{.cmd = "body_type", .u = "body_type [Search Criteria]", .fn = FindBodyType, .a = {"#findbodytype"}},
+		Cmd{.cmd = "bot", .u = "bot [Search Criteria]", .fn = FindBot, .a = {"#findbot"}},
```

Comment:
Can we append this bot command below `.emplace_back` style when the bots are enabled via rule?